Sources/General/ZLInputTextViewController.swift

```diff
@@ -213,7 +214,7 @@ extension ZLInputTextViewController: UICollectionViewDelegate, UICollectionViewD
 
 extension ZLInputTextViewController: UITextViewDelegate {
     func textView(_ textView: UITextView, shouldChangeTextIn range: NSRange, replacementText text: String) -> Bool {
-        if text == "\n" {
+        if !ZLImageEditorConfiguration.default().textStickerCanLineBreak && text == "\n" {
```

Comment:
I think the newlines of the text should be trimmed before the callback in the `doneBtnClick` method.

Reply:
@longitachi Thanks all for the lighting fast response! I fixed that in the `doneBtnClick` by `textView.text = textView.text.trimmingCharacters(in: .whitespacesAndNewlines)`.